addons/sale/sale.py

```diff
@@ -140,6 +140,24 @@ def _get_invoiced(self, cr, uid, ids, field_name, arg, context=None):
                 res[order.id]['invoiced'] = True
         return res
 
+    def _amount_total(self, cr, uid, context=None):
+        vals = {}
+        invoiced = 0.0
+        paid = 0.0
+        if context is None:
+            context = {}
+        order_id = context.get('active_id')
+        if order_id:
+            order = self.browse(cr, uid, order_id, context=context)
+            for invoice in order.invoice_ids:
+                if invoice.state not in ('cancel'):
+                    invoiced += invoice.amount_total
+                if invoice.state in ('paid'):
```

Comment:
it should be if invoice.state == 'paid':
here `in` operator with string operand lead to wrong result 
```python
>>> 'Good' in ('VeryGood')
True